playbook-website/config/webpack/environment.js

```diff
@@ -2,28 +2,56 @@ const path = require('path')
 
 const { environment } = require('@rails/webpacker')
 
-environment.loaders.insert('javascript', {
-  test: /\.(js|jsx)$/,
+const babelLoader = environment.loaders.get('babel')
+babelLoader.test = /\.(js|jsx|ts|tsx)?(\.erb)?$/
+
+const nodeLoader = environment.loaders.get('nodeModules')
+nodeLoader.exclude = [
+  nodeLoader.exclude,
+  /node_modules\/@powerhome\/playbook-icons-react/
+]
```

Comment:
This loader, again by default, attempts to make use of ES modules (`.mjs`) when it really shouldn't at least for this specific node module.